src/mmds/src/lib.rs

```diff
@@ -129,22 +121,18 @@ pub fn convert_to_response(request: Request) -> Response {
         );
     }
 
-    respond_to_request(request)
-}
-
-fn respond_to_request(request: Request) -> Response {
-    let mmds_version = MMDS.lock().expect("Poisoned lock").version();
+    let mut mmds_guard = mmds.lock().unwrap();
```

Comment:
Any particular reason for removing the unwrap() error here?

Reply:
I think you mean removing the expect, right?
Good point, reverted to an expect